paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/ResourceRepositoryModule.kt

```diff
@@ -0,0 +1,37 @@
+package com.stripe.android.paymentsheet.injection
+
+import android.content.Context
+import android.content.res.Resources
+import com.stripe.android.paymentsheet.address.AddressFieldElementRepository
+import com.stripe.android.paymentsheet.specifications.BankRepository
+import com.stripe.android.paymentsheet.specifications.ResourceRepository
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+/**
+ * Common module providing payment related dependencies.
+ * In order to use this module, [Context] need to be provided elsewhere.
+ */
+@Module
+internal class ResourceRepositoryModule {
```

Comment:
For dependencies whose constructor's parameters are all provided, you can just mark their constructor as `@Inject`  and not need to explicitly write `@Provides` method for them in a `@Module`.
In here, since `Resources` is provided, `provideBankRepository` and `provideAddressFieldRepository` are not needed, 
and since `BankRepository` and `AddressFieldElementRepository` are also provided, `provideResourceRepository` is not needed either :)

So this file can be deleted, please see the comments on the constructor of these three classes